tests/templates/kuttl/orphaned-resources/30-install-airflow-cluster.yaml.j2

```diff
@@ -26,8 +26,12 @@ metadata:
   name: airflow
 spec:
   image:
+{% if test_scenario['values']['airflow-latest'].find(",") > 0 %}
+    custom: "{{ test_scenario['values']['airflow-latest'].split(',')[1] }}"
+    productVersion: "{{ test_scenario['values']['airflow-latest'].split(',')[0] }}"
+{% else %}
     productVersion: "{{ test_scenario['values']['airflow-latest'] }}"
-    pullPolicy: IfNotPresent
```

Comment:
Was the removal of this line intended?

Reply:
reverted https://github.com/stackabletech/airflow-operator/pull/465/commits/c0e11419aaebc1c98a604cc56c8a9db6a68ecb55